src/ListBox/README.md

```diff
@@ -0,0 +1,132 @@
+# ListBox
+
+Provides a ListBox component that takes ListBoxOption components as children and handles the selection of and navigation through the ListBoxOption components.
+
+## API
+
+### `children` (ListBoxOption component array; required)
```

Comment:
Nit: It's not an `array` though? `ListBox`'s `children` should consist of zero or more `ListBoxOption` components, right?

Reply:
Are you planning on throwing any errors or warnings if an API user doesn't use `ListBoxOption`s for all the `children`?

Reply:
Yes. But the majority of Paragon components that explicitly specify the children prop refer to an array, with the exception being Fieldset, which specifies node. I'd like to make that consistent across the components.